### DISCUSSION TITLE
v4.1.92.0

### DISCUSSION THREAD

**Discussion by doug24** [dnGrep/dnGrep#1085] [General]
[![v4.1.92.0](https://img.shields.io/github/downloads/dnGrep/dnGrep/v4.1.92.0/total.svg)](https://github.com/dnGrep/dnGrep/releases/tag/v4.1.92.0)

## Changes in this release:

#1076 **.gitignore**  Fixed issue where the .git folder is not always excluded when `Use .gitignore` flag is checked.

### Other updates
This release contains an update to:
 - .NET v8.0.4

**Updated Translations** Three new languages have been started: Polish, Slovak, and Ukrainian. Thank-you translators! There are now 23 translations, but many of the languages need help to complete them. You are welcome and encouraged to complete or improve any of the existing translations, see [Translate dnGrep](https://github.com/dnGrep/dnGrep/wiki/Translate-dnGrep) for more information. Do as much or as little as you like. If you do not want to use Weblate for translations, I will take input in any format and merge the changes. Send me a note in the issues or discussions page.

### Checksum information

Name: dnGREP.4.1.92.x64.msi
SHA-256: 505388dc33e595a470314be71e3f2d5b16c4ee07665de2b9a90a275fc746f340

Name: dnGREP.4.1.92.x86.msi
SHA-256: ebca9f3adc18fd65468c461bb5a5f7d629a58194053ed058683bdc86f5db72d5

Name: dnGrep.4.1.92.0.x64.zip
SHA256: f045f62e3fae49fb04e3fc32077868200bbc45d63ee3445971ea95e0cf27a661

Name: dnGrep.4.1.92.0.x86.zip
SHA256: de26a6f0d60fcd89959d42b2e021509c29f8dd687087576cffdc311f33dcd700

This program uses free code signing provided by [SignPath.io](https://signpath.io?utm_source=foundation&utm_medium=github&utm_campaign=dngrep), and a certificate by the [SignPath Foundation](https://signpath.org?utm_source=foundation&utm_medium=github&utm_campaign=dngrep).

<hr /><em>This discussion was created from the release <a href='https://github.com/dnGrep/dnGrep/releases/tag/v4.1.92.0'>v4.1.92.0</a>.</em>

**Comment by doug24** [comment]
I'm confused - the Theme Editor is not installed with dnGrep. If you find it there, it must have been something you did.  I have not yet added the version to the Theme Editor - I know you asked for it - it is easy, but it hasn't been a priority.  I have been very busy working on the shell context menu for dnGrep. It didn't get done for the April release but should make it into the next release.

**Reply by DerellLicht** [reply]
Hah!!!  okay, I can see how that happened... yeah, I *had* put that link in the Programs folder... Apparently I tricked myself!!
Nevermind!!  I deleted my comment, just not relevant!!

And dnGrep is looking fantastic!!  Thanks again for all your work...
